### PR TITLE
Draw move ghost

### PR DESCRIPTION
**Pull request by eli7890**
The Ghost class was created and movement functions were added.

### CODE REVIEW

File: src/main/java/pacman/Main.java [hunk 1]
```diff
@@ -1,33 +1,53 @@
...
+
+
 class Main extends Frame{
 
     //attrib
-    private pacman p;
-    private food f;
-    private ghost g;
-    private ArrayList <wall> w; 
+    //private pacman p;
```
Comment (julia-jpg) [inline src/main/java/pacman/Main.java]: delete this variables if they aren't necessary

File: src/main/java/pacman/Main.java [hunk 2]
```diff
@@ -1,33 +1,53 @@
...
+    //private food f;
+    //private ghost g;
+    //private ArrayList <wall> w; 
 
     public Main(){
         this.setLayout(null);
-        this.setSize(200,200);
+        this.setSize(900,900);
 		this.setVisible(true);
```
Comment (julia-jpg) [inline src/main/java/pacman/Main.java]: fix the indentation

File: src/main/java/pacman/Main.java [hunk 3]
```diff
@@ -1,33 +1,53 @@
...
-
+        //JPanel panel = new JPanel();
```
Comment (julia-jpg) [inline src/main/java/pacman/Main.java]: delete these lines comments

File: src/main/java/pacman/Main.java [hunk 4]
```diff
@@ -1,33 +1,53 @@
...
+        //JLabel label = new JLabel(ghost);
+        //panel.add(label);
        // w = new ArrayList <wall>();     //create arraylist for outer wall
         //fill outer wall with -
         //Initialize object wall
         
     }
-
+    public void paint(Graphics g,int x, int y){
+        //g.setColor(Color.RED);
```
Comment (julia-jpg) [inline src/main/java/pacman/Main.java]: delete this line if isn't necessary

File: src/main/java/pacman/Main.java [hunk 5]
```diff
@@ -1,33 +1,53 @@
...
+        ImageIcon ghost = new ImageIcon("images/ghost.jpg");
+        Image ghostImg = ghost.getImage();
+        g.drawImage(ghostImg,0,0,50,50,null);
+
```
Comment (julia-jpg) [inline src/main/java/pacman/Main.java]: this space isn't necessary

File: src/main/java/pacman/Main.java [hunk 6]
```diff
@@ -1,33 +1,53 @@
...
-
+    public void paint(Graphics g,int x, int y){
+        //g.setColor(Color.RED);
+        ImageIcon ghost = new ImageIcon("images/ghost.jpg");
+        Image ghostImg = ghost.getImage();
+        g.drawImage(ghostImg,0,0,50,50,null);
+        
+    }
     public static void main(String[] args){
         //p = new pacman(8, 8, true);
```
Comment (julia-jpg) [inline src/main/java/pacman/Main.java]: delete this variables if they aren't necessary

File: src/main/java/pacman/Main.java [hunk 7]
```diff
@@ -1,33 +1,53 @@
...
         //f = new food(2, 1, true);
         //g = new ghost (3, 1, true);
         //w = new wall (4, 1, true);
-        
+        ghost ghost1 = new ghost(1,1,true);
+        Thread ghostThread = new Thread(ghost1);
+        ghostThread.start();
+        //Graphics g = new Graphics();
```
Comment (julia-jpg) [inline src/main/java/pacman/Main.java]: delete this if isn't necessary

File: src/main/java/pacman/Main.java [hunk 8]
```diff
@@ -1,33 +1,53 @@
...
+        paint(ghost1.getX(), ghost1.getY());
+         
         Main m = new Main(); 
+        
     }
-
```
Comment (julia-jpg) [inline src/main/java/pacman/Main.java:23]: this space isn't necessary

File: src/main/java/pacman/Main.java [hunk 9]
```diff
@@ -1,33 +1,53 @@
...
         //w = new wall (4, 1, true);
-        
+        ghost ghost1 = new ghost(1,1,true);
+        Thread ghostThread = new Thread(ghost1);
+        ghostThread.start();
+        //Graphics g = new Graphics();
+        paint(ghost1.getX(), ghost1.getY());
+         
         Main m = new Main(); 
+
```
Comment (julia-jpg) [inline src/main/java/pacman/Main.java]: this space isn't necessary

File: src/main/java/pacman/ghost.java [hunk 10]
```diff
@@ -1,7 +1,55 @@
-class ghost extends position {
-    private boolean eatable;
+class ghost extends position implements Runnable{
```
Comment (julia-jpg) [inline src/main/java/pacman/ghost.java]: Use uppercase at the beginning of name class ghost

File: src/main/java/pacman/ghost.java [hunk 11]
```diff
@@ -1,7 +1,55 @@
...
-		super(x,y,exist);
-        this.eatable = false;
+	 super(x,y,exist);
+			this.eatable = false;
 	}
-}
\ No newline at end of file
+	
+	public boolean getExist(){
+	 return exist;
```
Comment (julia-jpg) [inline src/main/java/pacman/ghost.java]: use indentation

File: src/main/java/pacman/ghost.java [hunk 12]
```diff
@@ -1,7 +1,55 @@
...
+	}
+	public void setX(int x){
+	 super.x = x;
```
Comment (julia-jpg) [inline src/main/java/pacman/ghost.java]: use indentation

File: src/main/java/pacman/ghost.java [hunk 13]
```diff
@@ -1,7 +1,55 @@
...
+	}
+	public void setY(int y){
+	 super.y = y;
```
Comment (julia-jpg) [inline src/main/java/pacman/ghost.java]: use indentation

File: src/main/java/pacman/ghost.java [hunk 14]
```diff
@@ -1,7 +1,55 @@
...
+	}
+	public boolean setExist(boolean exist){
+	 super.exist = exist;
```
Comment (julia-jpg) [inline src/main/java/pacman/ghost.java]: use indentation

File: src/main/java/pacman/ghost.java [hunk 15]
```diff
@@ -1,7 +1,55 @@
...
+	 return true;
+	}
+	public void moveUp(){
+	 super.y--;
```
Comment (julia-jpg) [inline src/main/java/pacman/ghost.java]: use indentation

File: src/main/java/pacman/ghost.java [hunk 16]
```diff
@@ -1,7 +1,55 @@
...
+	}
+	public void moveDown(){
+	 super.y++;
```
Comment (julia-jpg) [inline src/main/java/pacman/ghost.java]: use indentation

File: src/main/java/pacman/ghost.java [hunk 17]
```diff
@@ -1,7 +1,55 @@
...
+	}
+	public void moveLeft(){
+	 super.x--;
```
Comment (julia-jpg) [inline src/main/java/pacman/ghost.java]: use indentation

File: src/main/java/pacman/ghost.java [hunk 18]
```diff
@@ -1,7 +1,55 @@
...
+	}
+	public void moveRight(){
+	 super.x++;
```
Comment (julia-jpg) [inline src/main/java/pacman/ghost.java]: use indentation

File: src/main/java/pacman/ghost.java [hunk 19]
```diff
@@ -1,7 +1,55 @@
...
+		}
+	}
+	@Override
+	public void run(){
+		while(exist){
+			searchRoute();
+		}
+	}
+	public void stopRun(){
+			exist = false;
```
Comment (julia-jpg) [inline src/main/java/pacman/ghost.java]: correct indentation

**Comment by J0c4** [comment]
build failing
image

**Pull request merged**